fe/fe-core/src/main/java/com/starrocks/mv/MVRepairHandler.java

```diff
@@ -88,7 +88,7 @@ default void handleMVRepair(TransactionState transactionState) {
         }
 
         for (TableCommitInfo tableCommitInfo : transactionState.getIdToTableCommitInfos().values()) {
-            Table table = db.getTable(tableCommitInfo.getTableId());
+            Table table = GlobalStateMgr.getCurrentState().getLocalMetastore().getTable(db.getId(), tableCommitInfo.getTableId());
```

Comment:
need to make this `GlobalStateMgr.getCurrentState().getLocalMetastore()` as static method